docker/entrypoint

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh -e
+
+if ! whoami &>/dev/null; then
+  if [ -w /etc/passwd ]; then
+    echo "${USER_NAME:-push_proxy}:x:$(id -u):$(id -g):${USER_NAME:-push_proxy} user:${HOME}:/sbin/nologin" >> /etc/passwd
+  fi
+fi
```

Comment:
Same question as above. If we can automatically create the user, is there still a need for this ?